src/pages/chat.tsx

```diff
@@ -23,32 +54,44 @@ const ChatPage: NextPage = (props): JSX.Element => {
                             <div className="flex flex-col leading-tight">
                                 <div className="text-2xl mt-1 flex items-center">
                                     <span className="text-gray-700 mr-3">
-                                        StinkyGiraffe#7468
+                                        {session.user?.email?.split('@')[0]}
```

Comment:
I'll need to make a new ticket to implement the username into the token.

Reply:
@piyasamyak Sorry, I got this wrong, the username is returned when logging in. See [`auth.controller.ts`](https://github.com/Project-SAP/project-sap-backend/blob/6a929e772871baf6aa521b5e2deeff962e4082a1/src/controller/auth.controller.ts#L122) in the backend for verification of this.

But the username can be added to the session and inserted here.